io/eolearn/tests/test_processing_io.py

```diff
@@ -1,28 +1,23 @@
 """ Testing SentinelHubInputTask
 """
-
 import os
 import shutil
 import unittest
 from concurrent import futures
 import datetime as dt
-import numpy as np
-from sentinelhub import CRS, BBox, DataSource
 
+import numpy as np
+from sentinelhub import CRS, BBox, DataCollection
 from eolearn.io import SentinelHubInputTask, SentinelHubDemTask
-from eolearn.core import FeatureType
+from eolearn.core import FeatureType, EOPatch
+
+from test_io import IoTestCase
 
-# import sys
-# import logging
-# logging.basicConfig(stream=sys.stdout)
-# logging.getLogger("eolearn.io.processing_api").setLevel(logging.DEBUG)
-# logging.getLogger("sentinelhub.sentinelhub_client").setLevel(logging.DEBUG)
-# logging.getLogger("sentinelhub.sentinelhub_rate_limit").setLevel(logging.DEBUG)
 
 def array_stats(array):
     time, height, width, _ = array.shape
     edge1 = np.mean(array[int(time/2):, 0, 0, :])
-    edge2 = np.mean(array[:int(time/2), -1, -1, :])
+    edge2 = np.mean(array[:max(int(time/2), 1), -1, -1, :])
```

Comment:
Could you shortly explain why `max` here?

Reply:
If `time=1` then `array[:int(time/2), -1, -1, :]` would be an empty array and `np.mean` on an empty array returns NaN with a warning.

In the new tests such a case appears so I updated this in a way that wouldn't break previous test cases.